src/checks.rs

```diff
@@ -2076,6 +2076,7 @@ impl CheckKind {
                 | CheckKind::SectionUnderlineAfterName(..)
                 | CheckKind::SectionUnderlineMatchesSectionLength(..)
                 | CheckKind::SectionUnderlineNotOverIndented(..)
+                | CheckKind::SetAttrWithConstant
```

Comment:
@harupy - Small thing -- these need to be added to this `fixable` list once they support autofix. That way, they'll automatically update properly when you run `cargo dev generate-rules-table`.